commcare_export/builtin_queries.py

```diff
@@ -22,22 +23,27 @@ def mapped_source_field(self):
         if not self.map_function:
             return Reference(self.source)
         else:
-            return Apply(Reference(self.map_function), Reference(self.source),
-                         *self.extra_args)
+            return Apply(
+                Reference(self.map_function), Reference(self.source),
+                *self.extra_args
+            )
 
 
 def compile_query(columns, data_source, table_name):
-    source = Apply(Reference('api_data'), Literal(data_source),
-                   Reference('checkpoint_manager'))
-    part = excel_query.SheetParts(table_name, [c.name for c in columns], source,
-                                  List([c.mapped_source_field for c in columns]),
-                                  None)
+    source = Apply(
+        Reference('api_data'), Literal(data_source),
+        Reference('checkpoint_manager')
+    )
+    part = excel_query.SheetParts(
+        table_name, [c.name for c in columns], source,
+        List([c.mapped_source_field for c in columns]), None
```

Comment:
Same here. Per my aesthetic preferences
```
this_style(is_for, cramming, as_many, args,
           on, each, line, as_is, possible)

# and

this_other_better_style(
    puts_each,
    argument,
    on_a,
    separate,
    line,
)
```
When I see the second, I expect each line to be a separate argument, so what YAPF is doing here is a little unexpected.

I forget if YAPF has an option for that?

Reply:
Yes! Great suggestion. See https://github.com/dimagi/commcare-export/pull/207/commits/f7ff584d5eb5640f734fa016c1fade20feb18a18

You have to be a bit careful, cos that setting can mess with comprehensions:
```python
    {foo: bar for foo, bar in some_dictionary.items() if do_i_want_this_one(bar)}
```
becomes
```python
    {foo: bar for foo,
     bar in some_dictionary.items() if do_i_want_this_one(bar)}
```

But if you unpack into a tuple, it does a better job:
```python
    {foo: bar for (foo, bar) in some_dictionary.items() if do_i_want_this_one(bar)}
```
becomes
```python
    {foo: bar for (foo, bar) in some_dictionary.items()
     if do_i_want_this_one(bar)}
```